shared/src/commonMain/kotlin/org/hyperskill/app/track_selection/details/presentation/TrackSelectionDetailsReducer.kt

```diff
@@ -104,15 +108,43 @@ internal class TrackSelectionDetailsReducer : StateReducer<State, Message, Actio
     ): ReducerResult =
         state.copy(isTrackLoadingShowed = false) to
             when (message) {
-                TrackSelectionDetailsFeature.TrackSelectionResult.Success ->
+                TrackSelectionDetailsFeature.TrackSelectionResult.Success -> {
+                    val navigationAction = when {
+                        !state.isNewUserMode ->
+                            ViewAction.NavigateTo.StudyPlan
+                        isProjectSelectionAvailable(state) ->
+                            ViewAction.NavigateTo.ProjectSelectionList(
+                                trackId = state.trackWithProgress.track.id,
+                                isNewUserMode = state.isNewUserMode
+                            )
+                        else ->
+                            ViewAction.NavigateTo.Home
```

Comment:
In what cases `ViewAction.NavigateTo.Home` will be called? I suppose that could be called for a new user and not, so on iOS it's impossible to navigate to home without more information because a tab bar controller could not exist.

If so I think it's better to provide `ViewAction.NavigateTo.Home` for when `!state.isNewUserMode` and `ViewAction.NavigateTo.NewRootHome` when `state.isNewUserMode`

Reply:
Or do like with `ProjectSelectionDetailsFeature.Action.ViewAction.NavigateTo.StudyPlan` here we have `NavigationCommand`

Reply:
`ViewAction.NavigateTo.Home` is sent in case the user is new and project selection is not available at the moment.
https://github.com/hyperskill/mobile-app/blob/52114be96dc1d52a20c8a85a9c0b1006a8a88fad/shared/src/commonMain/kotlin/org/hyperskill/app/track_selection/details/presentation/TrackSelectionDetailsReducer.kt#L112-L122

Reply:
@ivan-magda 
I've added a navigation command to the `NavigateTo.Home`.
Please, support it on the iOS side.

Reply:
Thanks, I'll support in scope of the ALTAPPS-801